index.js

```diff
@@ -20,9 +20,23 @@ module.exports = function (content) {
       const compiled = compiler.compile(result.data, {
         preserveWhitespace: false,
       });
+      
+      const transpileCode = `var render = function (${options.functional ? '_h,_vm' : ''}) { ${compiled.render} };`;
 
-      component = transpile(`var render = function () {${compiled.render}};`);
-      component += `module.exports = { render: render };`;
+      const transpileOptions = {
+        transforms: {
+          stripWith: true,
+          stripWithFunctional: options.functional || false
```

Comment:
Please coerce the `options.functional` to boolean using for example two exclamation marks and add the comma at the end of the line to be consistent with the rest of code:
```js
 stripWithFunctional: !!options.functional || false,
```